packages/devtools_app/lib/src/screens/profiler/cpu_profiler.dart

```diff
@@ -179,6 +179,21 @@ class _CpuProfilerState extends State<CpuProfiler>
               const SizedBox(width: denseSpacing),
               UserTagDropdown(widget.controller),
               const SizedBox(width: denseSpacing),
+              ValueListenableBuilder<bool>(
+                valueListenable: preferences.vmDeveloperModeEnabled,
+                builder: (context, vmDeveloperModeEnabled, _) {
+                  if (!vmDeveloperModeEnabled) {
+                    return Container();
+                  }
+                  return Row(
+                    children: [
+                      ModeDropdown(widget.controller),
+                      const SizedBox(width: denseSpacing),
```

Comment:
wrap ModeDropdown in a Padding widget with right padding instead of using a Row here. That will make it more clear what the sized box is for.